code_samples/data_migration/examples/create_attribute_group.yaml

```
@@ -0,0 +1,24 @@
+-   type: attribute_group
+    mode: create
+    identifier: hat
+    names:
+        eng-GB: Hat
+-   type: attribute
+    mode: create
+    identifier: size
+    attribute_type_identifier: integer
+    attribute_group_identifier: hat
+    names:
+        eng-GB: Size
+-   type: attribute
+    mode: create
+    identifier: color
+    attribute_type_identifier: selection
+    attribute_group_identifier: hat
+    names:
+        eng-GB: Color
+    options:
+        choices:
+            - { value: red, label: { "eng-GB": "Red"}}
```

Just an internal note, but this might not be obvious:

```suggestion
            -    value: red
                 label: 
                     "eng-GB": "Red"
```
is also a valid notation (which may provide better readability in some cases, especially with long data).